src/generate.js

```diff
@@ -192,21 +192,23 @@ const runStringHandlers = (
 ) /* */ => {
     const result = {};
 
-    Object.keys(declarations).forEach(key => {
+    const hasStringHandlers = !!stringHandlers;
+    const keys = Object.keys(declarations);
+    for (var i = 0; i < keys.length; i += 1) {
```

Comment:
Oops! Should be the same speed, since I think babel compiles it down to `var`. I think all of the error checking related to the block scoping happens at build time.